SurfaceTopography/NonuniformLineScan.py

```diff
@@ -188,7 +188,7 @@ def height_scale_factor(self):
 
     # For backwards compatibility
     @property
-    @deprecated(version='0.94.0', reason='Use heights_scale_factor')
+    @deprecated(version='0.94.0', alternative='heights_scale_factor')
```

Comment:
The property in the code is called `height_scale_factor`, without "s".
There are more places affected by this (search for "heights_scale_factor").